Tests/JWSRSATests.swift

```diff
@@ -149,7 +128,19 @@ class JWSRSATests: RSACryptoTestCase {
         let verifier = Verifier(verifyingAlgorithm: algorithm, publicKey: publicKeyAlice2048!)
 
         XCTAssertTrue(secondJWS.isValid(for: verifier!))
-        XCTAssertEqual(String(data: jws.header.data(), encoding: .utf8), "{\"alg\":\"\(algorithm.rawValue)\"}")
+        if withKid {
```

Comment:
This if statement wants to be a guard. If you move the check from line 144 up above the branch then an early return wouldn't matter, right?

Reply:
Updated ✅